lib/component-preprocessor/themes-templates.js

```diff
@@ -31,9 +34,6 @@ module.exports = function(tree, themeName, options) {
     symlinkOrCopySync(src, target);
   });
 
-  // if we determine we need to auto generate because there is no component for the
-  // scss file, but there is a template, let the original template win in the working tree
-  workingTree = mergeTrees([generatedComponents, workingTree], { overwrite: true });
-
+  workingTree = mergeTrees([workingTree, generatedComponents]);
```

Comment:
really happy to get rid of this comment and the need for it, now we know for sure that the generated template for sure belongs there